File: docs/src/pages/data-shaping/snippets/filtering/default.py
app.layout = DashSpreadGrid(  # type: ignore
    data={
        # collapse: true
        # default dict data
        # collapse: false
    },
    # collapse: true
    columns=[{"type": "DATA-BLOCK", "width": 70}],
    # collapse: false
    rows=[
        {"type": "HEADER"},
        {"type": "FILTER"},
        {"type": "DATA-BLOCK"},
    ],
    filters=[{"columnId": "registered", "expression": True}],
)

File: docs/src/pages/data-shaping/snippets/sorting/default.js
function render(sortBy) {
    SpreadGrid(div, {
        data: [
            // collapse: true
            // default data
            // collapse: false
        ],
        // collapse: true
        columns: [{ type: 'DATA-BLOCK', width: 70 }],
        // collapse: false
        sortBy: sortBy,
        onSortByChange: render
    });
}

render([{ columnId: "age", direction: "ASC" }]);